integration_test/third_party_apps_data/applications/elasticsearch/metadata.yaml

```diff
@@ -131,6 +131,163 @@ expected_metrics:
     labels:
       state: .*
       thread_pool_name: .*
+  - type: workload.googleapis.com/elasticsearch.breaker.memory.estimated
+    value_type: INT64
+    kind: GAUGE
+    monitored_resource: gce_instance
+    labels:
+      name: .*
```

Comment:
In the scope doc this was `circuit_breaker_name` but here it's just `name`. I noticed most of the labels in this integration do that sort of shorten. Why did we do that instead of matching it to the scope doc? (or vice versa, why didn't the scope doc reflect these names)

Reply:
The scope doc had the attributes listed names, but not the actual value which has been corrected. So for example in otel metadata.yaml, [`elasticsearch.breaker.memory.estimated`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/elasticsearchreceiver/metadata.yaml#L142-L147) metric has an attribute listed `circuit_breaker_name` which actualy has a value of [`name`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/elasticsearchreceiver/metadata.yaml#L84-L85). The ops agent metadata.yaml is correct and the [scope doc](https://docs.google.com/document/d/1ByUTgrwwbL5ihuqb7QTE_XZMRE7aAfHS6KBsD8_n_l0/edit) has been corrected so that `circuit_breaker_name` -> `name` along with other metrics.

Reply:
Thanks for making that update. Looks good.